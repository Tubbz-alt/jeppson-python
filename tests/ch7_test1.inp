    7    2  100  1.217E-5   0.0001   64.4    
    1  12.0       3.0E3     1.2E-2    1.67
    2  14.0       2.0E3     1.2E-2    3.45
    3  10.0       3.0E3     1.2E-2    1.0
    4  10.0       2.0E3     1.2E-2    1.67
    5  14.0       3.0E3     1.2E-2    1.78
    6  10.0       2.0E3     1.2E-2    1.56
    7  12.0       3.0E3     1.2E-2    0.67
    4    1    2   -3   -4
    4   -2   -5    6    7
