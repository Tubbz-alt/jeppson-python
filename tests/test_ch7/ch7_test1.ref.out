Pipe    Diameter         Length           Rel. Roughness
  0     1.2000E+01 in    3.0000E+03 ft    1.0000E-03
  1     1.4000E+01 in    2.0000E+03 ft    8.5714E-04
  2     1.0000E+01 in    3.0000E+03 ft    1.2000E-03
  3     1.0000E+01 in    2.0000E+03 ft    1.2000E-03
  4     1.4000E+01 in    3.0000E+03 ft    8.5714E-04
  5     1.0000E+01 in    2.0000E+03 ft    1.2000E-03
  6     1.2000E+01 in    3.0000E+03 ft    1.0000E-03


Final results:

Pipe  Flow                     Flow                      Flow                    Head Loss       Head Loss
  0   4.9481E-02 m ** 3 / s    1.7474E+00 ft ** 3 / s    7.8429E+02 gal / min    1.4879E+00 m    4.8817E+00 ft
  1   9.8750E-02 m ** 3 / s    3.4873E+00 ft ** 3 / s    1.5652E+03 gal / min    1.7244E+00 m    5.6573E+00 ft
  2   2.6125E-02 m ** 3 / s    9.2260E-01 ft ** 3 / s    4.1409E+02 gal / min    1.1007E+00 m    3.6113E+00 ft
  3   4.5098E-02 m ** 3 / s    1.5926E+00 ft ** 3 / s    7.1481E+02 gal / min    2.1116E+00 m    6.9278E+00 ft
  4   4.9270E-02 m ** 3 / s    1.7399E+00 ft ** 3 / s    7.8094E+02 gal / min    6.7222E-01 m    2.2054E+00 ft
  5   4.5309E-02 m ** 3 / s    1.6001E+00 ft ** 3 / s    7.1816E+02 gal / min    2.1309E+00 m    6.9913E+00 ft
  6   2.0107E-02 m ** 3 / s    7.1006E-01 ft ** 3 / s    3.1870E+02 gal / min    2.6564E-01 m    8.7151E-01 ft

